pymess/management/commands/sync_emails.py

```diff
@@ -34,3 +31,15 @@ def handle(self, *args, **options):
             self.stdout.write(
                 self.style.SUCCESS('Synced "{}" e-mail templates'.format(len(files_with_email_html_body)))
             )
+
+    def handle(self, *args, **options):
+        verbosity = int(options.get('verbosity'))
+        if verbosity > 0:
+            self.stdout.write('Syncing e-mails')
+
+        directory = os.path.join(settings.EMAIL_HTML_DATA_DIRECTORY)
+        for filename in os.listdir(directory):
```

Comment:
`filename` is actually a directory name (or should be), specifically the locale directory name. Maybe call it `locale_directory`?

Reply:
I've added comments to clarify my naming. The root dir may contain `.html` files that would be synced for a locale with `None` value.